skycoin/dmsg
# Improve dmsg-server CLI

Fixes #78 

 Changes:	
- add `start` and `config` command for `dmsg-server`
- add `gen` command for `config` with two flag:
  - `-o` for output file
  - `-t` for use test value of discovery URL

How to test this PR:
- Build on current branch `make build`
- Generate config by `./bin/dmsg-server config gen -o dmsgconfig.json` and `./bin/dmsg-server config gen -t` and check related configs
- Start `dmsg-server` by `./bin/dmsg-server start dmsgconfig.json` or `./bin/dmsg-server start config.json`
